src/components/results.js

```diff
@@ -89,10 +89,27 @@ const ResultsComponent = React.createClass({
           block.userId
         }/${d._id}/raw/${block.thumb}/thumbnail.png)`
       } else if (screenshots.indexOf(`${d._id}.png`) > -1) {
-        style.backgroundImage = `url(https://christopheviau.com/block_screenshot/${
-          d._id
-        }.png)`
-        classNameString = `${classNameString} ` + `no-thumbnail`
+        const chrisVStem = 'https://christopheviau.com'
+        const chrisVUrl = `${chrisVStem}/block_screenshot/${d._id}.png`
```

Comment:
Definitely cleaner.

Reply:
yea, and each part is on a single line 😄